libcxx/include/map

```diff
@@ -702,6 +703,41 @@ public:
 #  endif
 };
 
+#  if _LIBCPP_STD_VER >= 14
+template <class _CP, class _Key, class _Compare>
```

Comment:
I am not sure what `_CP` stands for, but I find it confusing. I'd suggest renaming to something more telling, like `_MapValueType`, which makes it clear that we're operating on a `value_type == pair<key_type const, mapped_type>`.